jgrapht-core/src/main/java/org/jgrapht/alg/isomorphism/GraphOrdering.java

```diff
@@ -39,7 +39,13 @@
 
     private int[][] outgoingEdges;
     private int[][] incomingEdges;
-    private Boolean[][] adjMatrix;
+    /*
+     * if caching is enabled, this contains cached information on edges existing, values:
```

Comment:
Convert this into a Javadoc comment and use proper formatting (unordered list HTML tags).

Reply:
done.